sender_otel_honey.go

```diff
@@ -106,3 +130,39 @@ func (t *SenderOTel) CreateSpan(ctx context.Context, name string, level int, fie
 	ots.Span = span
 	return ctx, ots
 }
+
+func setupOTELHTTPClient(opts *Options) otlptrace.Client {
```

Comment:
nonblocking: this would be cleaner and more consistent with common usage as setupOTelHTTPClient; same with the other function